### DISCUSSION TITLE
PYG=2.0.2 in Conda

### DISCUSSION THREAD

**Discussion by shjiang-yang** [pyg-team/pytorch_geometric#3407] [Q&A]
Hello, Why can't I use conda to install the latest version of PYG(2.0.2). The 2.0.1 version of PYG is always installed. My Pytorch version is 1.8. thanks.

**Comment by rusty1s** [comment]
Conda packages for PyG 2.0.2 are only provided for PyTorch 1.9.0 and PyTorch 1.10.0, as we aim to only support for the two latest PyTorch releases.
